### DISCUSSION TITLE
Pine Ebook Reader.

### DISCUSSION THREAD

**Discussion by anathonous** [johnfactotum/foliate#857] [Ideas]
You should modify foliate to run on full screen interfaces like the new pine ereader. Would be cool to install Gentoo on it with a version of x that auto launched foliate.

**Comment by johnfactotum** [comment]
How do you mean? Can't you already run it full screen?

**Comment by anathonous** [comment]
You should fork it so the library ui runs full screen so it could auto exec on Linux ereaders. Without having window decorations.

**Reply by johnfactotum** [reply]
Right, the library view can't be fullscreened. That will be fixed in the future when #509 is fixed.

But I think for your use case, you can achieve a similar effect by simply maximizing the window.
